Updating Functions

Neodevils opened the pull request:
- [ ] Added / Updated Functions on server channel.

Commit:
Updated $random.

Commit:
Merge branch 'aoijs:v5' into neo-main

USERSATOSHI commented on src/functions/usdrandom.md:
```diff
@@ -1,20 +1,48 @@
+  $random[1;10;yes;yes]
+  $random[1;10;yes;yes]
+  `
+  // Returns two decimal numbers that different, between 1 and 10
```
*that are different

you missed an `are`

Commit:
Update usdrandom.md

Fixed typo.

Commit:
Merge branch 'aoijs:v5' into neo-main

Commit:
Update usdsettimeout.md

Commit:
Merge branch 'aoijs:v5' into neo-main

Commit:
Delete usdsongfilter.md

Commit:
Added $pauseTrack

Commit:
Added $resumeTrack.

Commit:
Changed link on labels on one of question.

Commit:
Updated $sendWebhookMessage

Commit:
Added $nonEscape.

Commit:
Updated embed note → footnote.

Commit:
Updated $stop.

Commit:
Updated $skip & $stop.

Commit:
Added $setFilter, Updated $addFilter.

Commit:
Deleted $userTyping, not in summary but on website somehow.

Commit:
Merge branch 'aoijs:v5' into neo-main

Commit:
Merge branch 'aoijs:v5' into neo-main

Commit:
Merge branch 'v5' into neo-main

ImgBotApp committed:
[ImgBot] Optimize images

*Total -- 2,680.65kb -> 1,858.91kb (30.65%)

/.gitbook/assets/image (69).png -- 47.36kb -> 21.40kb (54.81%)
/.gitbook/assets/image (70).png -- 29.33kb -> 14.30kb (51.25%)
/.gitbook/assets/image (49).png -- 57.77kb -> 28.53kb (50.61%)
/.gitbook/assets/image.png -- 51.72kb -> 25.56kb (50.59%)
/.gitbook/assets/image (72).png -- 50.20kb -> 25.74kb (48.72%)
/.gitbook/assets/image (58).png -- 17.84kb -> 9.50kb (46.79%)
/.gitbook/assets/image (43).png -- 15.25kb -> 8.23kb (46.03%)
/.gitbook/assets/image (51).png -- 9.75kb -> 5.28kb (45.82%)
/.gitbook/assets/image (37).png -- 87.16kb -> 48.11kb (44.8%)
/.gitbook/assets/image (66).png -- 113.73kb -> 66.85kb (41.22%)
/.gitbook/assets/image (19).png -- 103.48kb -> 60.91kb (41.14%)
/.gitbook/assets/screenshot-2020-11-23-at-9.57.28-pm.png -- 7.74kb -> 4.56kb (41.09%)
/.gitbook/assets/image (35).png -- 184.69kb -> 113.25kb (38.68%)
/.gitbook/assets/screenshot-2020-11-23-at-10.00.16-pm.png -- 8.05kb -> 4.95kb (38.47%)
/.gitbook/assets/image (65).png -- 14.73kb -> 9.18kb (37.64%)
/.gitbook/assets/image (60).png -- 45.15kb -> 28.53kb (36.82%)
/.gitbook/assets/image (7).png -- 55.43kb -> 35.34kb (36.25%)
/.gitbook/assets/image (4).png -- 43.23kb -> 27.70kb (35.91%)
/.gitbook/assets/client-option-json.png -- 74.91kb -> 49.43kb (34.01%)
/.gitbook/assets/screenshot-2020-11-23-at-9.54.22-pm.png -- 2.15kb -> 1.42kb (34%)
/.gitbook/assets/image (56).png -- 9.88kb -> 6.62kb (32.98%)
/.gitbook/assets/image (68).png -- 9.94kb -> 6.87kb (30.93%)
/.gitbook/assets/image (38) (1).png -- 12.17kb -> 8.44kb (30.63%)
/.gitbook/assets/screenshot_820.png -- 67.54kb -> 47.34kb (29.91%)
/.gitbook/assets/image (29) (1) (1) (1) (2) (3) (2).png -- 41.50kb -> 29.18kb (29.69%)
/.gitbook/assets/image (29) (1) (1) (1) (2) (3) (2) (2).png -- 41.50kb -> 29.18kb (29.69%)
/.gitbook/assets/image (29) (1) (1) (1) (2) (3) (2) (3).png -- 41.50kb -> 29.18kb (29.69%)
/.gitbook/assets/image (29) (1) (1) (1) (2) (3) (2) (3) (1).png -- 41.50kb -> 29.18kb (29.69%)
/.gitbook/assets/image (32).png -- 46.51kb -> 33.29kb (28.43%)
/.gitbook/assets/image (50).png -- 44.37kb -> 31.77kb (28.41%)
/.gitbook/assets/image (28).png -- 14.23kb -> 10.24kb (28.07%)
/.gitbook/assets/image (64).png -- 5.37kb -> 3.86kb (28.05%)
/.gitbook/assets/image (47).png -- 6.55kb -> 4.73kb (27.79%)
/.gitbook/assets/image (39) (2) (2) (2) (3) (4).png -- 3.87kb -> 2.80kb (27.5%)
/.gitbook/assets/image (39) (2) (2) (2) (3).png -- 3.87kb -> 2.80kb (27.5%)
/.gitbook/assets/image (39) (2) (2) (2) (3) (3).png -- 3.87kb -> 2.80kb (27.5%)
/.gitbook/assets/image (39) (2) (2) (2) (3) (1).png -- 3.87kb -> 2.80kb (27.5%)
/.gitbook/assets/image (23).png -- 42.70kb -> 31.20kb (26.93%)
/.gitbook/assets/image (55).png -- 38.80kb -> 28.80kb (25.77%)
/.gitbook/assets/image (53).png -- 28.76kb -> 21.37kb (25.69%)
/.gitbook/assets/image (45).png -- 465.44kb -> 353.33kb (24.09%)
/.gitbook/assets/image (12).png -- 15.67kb -> 11.89kb (24.08%)
/.gitbook/assets/image (17).png -- 3.82kb -> 2.91kb (23.93%)
/.gitbook/assets/image (2).png -- 15.11kb -> 11.54kb (23.65%)
/.gitbook/assets/image (67).png -- 23.88kb -> 18.28kb (23.46%)
/.gitbook/assets/image (41).png -- 4.97kb -> 3.80kb (23.4%)
/.gitbook/assets/image (21) (1) (1) (1) (2) (3) (3) (3).png -- 15.00kb -> 11.52kb (23.17%)
/.gitbook/assets/image (21) (1) (1) (1) (2) (3) (3).png -- 15.00kb -> 11.52kb (23.17%)
/.gitbook/assets/image (21) (1) (1) (1) (2) (3) (3) (1).png -- 15.00kb -> 11.52kb (23.17%)
/.gitbook/assets/image (74).png -- 2.12kb -> 1.64kb (22.67%)
/.gitbook/assets/image (54).png -- 6.66kb -> 5.16kb (22.57%)
/.gitbook/assets/image (9).png -- 11.19kb -> 8.68kb (22.42%)
/.gitbook/assets/image (9) (1).png -- 11.19kb -> 8.68kb (22.42%)
/.gitbook/assets/image (5).png -- 119.07kb -> 92.48kb (22.33%)
/.gitbook/assets/image (20).png -- 8.09kb -> 6.29kb (22.2%)
/.gitbook/assets/image (25).png -- 7.84kb -> 6.12kb (22.01%)
/.gitbook/assets/image (13).png -- 14.02kb -> 10.94kb (21.98%)
/.gitbook/assets/image (16).png -- 3.92kb -> 3.09kb (21.21%)
/.gitbook/assets/image (1) (1) (1) (1) (1) (1) (1) (1).png -- 8.17kb -> 6.46kb (20.94%)
/.gitbook/assets/image (1) (1) (1) (1) (1) (1) (1).png -- 8.17kb -> 6.46kb (20.94%)
/.gitbook/assets/image (3).png -- 74.88kb -> 59.82kb (20.11%)
/.gitbook/assets/image (62).png -- 5.75kb -> 4.63kb (19.4%)
/.gitbook/assets/image (40).png -- 3.24kb -> 2.62kb (19.07%)
/.gitbook/assets/image (48).png -- 4.71kb -> 3.81kb (19.02%)
/.gitbook/assets/image (14) (4) (4) (3) (6).png -- 10.25kb -> 8.31kb (18.92%)
/.gitbook/assets/image (14) (4) (4) (3) (1).png -- 10.25kb -> 8.31kb (18.92%)
/.gitbook/assets/image (14) (4) (4) (3) (1) (1) (1).png -- 10.25kb -> 8.31kb (18.92%)
/.gitbook/assets/image (14) (4) (4) (3) (3) (1).png -- 10.25kb -> 8.31kb (18.92%)
/.gitbook/assets/image (14) (4) (4) (3).png -- 10.25kb -> 8.31kb (18.92%)
/.gitbook/assets/image (14) (4) (4) (3) (1) (1).png -- 10.25kb -> 8.31kb (18.92%)
/.gitbook/assets/image (14) (4) (4) (3) (3).png -- 10.25kb -> 8.31kb (18.92%)
/.gitbook/assets/image (14) (4) (4) (3) (5).png -- 10.25kb -> 8.31kb (18.92%)
/.gitbook/assets/image (14) (4) (4) (3) (1) (3).png -- 10.25kb -> 8.31kb (18.92%)
/.gitbook/assets/image (14) (4) (4) (3) (1) (2).png -- 10.25kb -> 8.31kb (18.92%)
/.gitbook/assets/image (14) (4) (4) (3) (4).png -- 10.25kb -> 8.31kb (18.92%)
/.gitbook/assets/image (18).png -- 6.79kb -> 5.55kb (18.38%)
/.gitbook/assets/image (8).png -- 8.20kb -> 6.74kb (17.82%)
/.gitbook/assets/image (27).png -- 4.37kb -> 3.59kb (17.77%)
/.gitbook/assets/image (15).png -- 50.53kb -> 41.82kb (17.23%)
/.gitbook/assets/image (44).png -- 47.57kb -> 39.52kb (16.93%)
/.gitbook/assets/image (10).png -- 2.90kb -> 2.41kb (16.63%)
/.gitbook/assets/mtt45fdb8q.png -- 20.04kb -> 18.03kb (10%)
/.gitbook/assets/image (57).png -- 1.63kb -> 1.52kb (6.48%)
/.gitbook/assets/image (73).png -- 8.28kb -> 7.85kb (5.27%)
/.gitbook/assets/image (24).png -- 26.67kb -> 25.99kb (2.57%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>

Commit:
Merge pull request #21 from Neodevils/imgbot

[ImgBot] Optimize images

Neodevils commented:
@USERSATOSHI still waiting for you 👻.
